designs/setdict-design.md

```diff
@@ -0,0 +1,104 @@
+# Set/Tuple/Dictionary: Project Milestone For Week 7
+For now, we'd like to start with dictionary. The following design document will be based on dictionary, but may be subject to change later.
+
+## Test Cases
+1. Initialize the empty dictionary in Chocopy.   
+```python
+dict_1 : [int, int] = None 

```

Comment:
This is not proper Python. I would look up the type hints Python uses to dictionaries and sets and stick to those, unless there's a reason you're deviating from Python's syntax.